spec/workers/comments/calculate_score_worker_spec.rb

```diff
@@ -15,12 +15,11 @@
         allow(BlackBox).to receive(:calculate_spaminess).and_return(99)
       end
 
-      it "updates score and spaminess_rating", :aggregate_failures do
+      it "updates score", :aggregate_failures do
```

Comment:
Quick suggestion: let's remove `aggregate_failures` from this `it` block since we are now only testing for a single expectation!: 
```suggestion
      it "updates the score" do
```